### ISSUE TITLE
The 'onSortingChange' event is not getting triggered from the SortingState plugin when performing server side sorting

### ISSUE THREAD

**Issue by SergeyAlexeev** [DevExpress/devextreme-reactive#1283]
- [x] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### I'm using ...

- [x] React Grid
- [ ] React Chart
- [ ] Vue Grid

### Environment

- devextreme-reactive: 1.1.7

**Comment by MaximKudriavtsev** [comment]
Hi,
 
I researched your scenario and couldn't reproduce the problem on my side.
 
We don't provide React Grid v1.1.7. The `npm view @devexpress/dx-react-grid versions` command shows all available versions. So, I'm using React Grid v1.1.2 for research.
 
First, I checked the scenario with remote sorting. It works as expected. This [sample](https://github.com/DevExpress/devextreme-reactive/files/2252881/react-grid-sorting.zip) shows this in action. Invoke the following commands to run it in a browser:
 
```
npm i
npm run dev
```
 
See the `README.md` file for more details.
 
Then I tested sorting with server side rendering. I didn't find any problem either. I used this [project](https://github.com/mui-org/material-ui/tree/master/examples/nextjs) for testing. I added React Grid to the [index.js](https://github.com/mui-org/material-ui/blob/master/examples/nextjs/pages/index.js) page and ran it. I should say, here I used React Grid v1.5.1, because React Grid v1.1.2 isn't compatible with the latest version of Materiau-UI that is used it the test project.
